rclpy/rclpy/executors.py

```diff
@@ -440,14 +460,15 @@ def _wait_for_ready_callbacks(
         :param timeout_sec: Seconds to wait. Block forever if ``None`` or negative.
             Don't wait if 0.
         :param nodes: A list of nodes to wait on. Wait on all nodes if ``None``.
+        :param condition: A callable that makes the function return immedaitely when it's True.
```

Comment:
@ivanpauno consider: 
```suggestion
        :param condition: A callable that makes the function return immediately when it evaluates to True.
```